internal/pkg/template/templates/addons/aurora/env/serverlessv2.yml

```diff
@@ -51,7 +48,7 @@ Resources:
       VpcId: !Ref VPCID
       Tags:
         - Key: Name
-          Value: !Sub 'copilot-${App}-${Env}-AuroraCluster'
```

Comment:
Because I thought they were unnecessary. However the other secuirty groups (e.g. EnvironmentSecurityGroup)'s tags also have the `copilot-` prefix. So I added them back.